### ISSUE TITLE
Time shift problem

### ISSUE THREAD

**Issue by sslupsky** [grafana/grafana#5935]
When you specify a time shift in the time range tab of the graph panel and then use the mouse the select a range to view, the time shift is applied to the mouse selection.  Thus, what is rendered is not what you selected to view.

**Comment by torkelo** [comment]
It is, as you use said the time shift is applied to selection, so it what you selected is what is shown, or did you mean something different?

**Comment by sslupsky** [comment]
Here are a few screen shots that explain what I mean.

In the first shot, I display a graph with some anomalous "events".  I click and drag using the mouse the event at the left hand side of the graph at time 1700h to "zoom" into the event.  In the second shot, you see the result which is the expected result.  That is, the timeline is centered at about 1700h and we have zoomed into the event.

In the third screen shot, I selected the event at 0800h near the right hand side of the graph.  This time, the time shift was applied to the graph of 2h.  When I click and drag the mouse around the 0800h event, you get what is shown in the forth screen shot.  This is the unexpected result.  That is, the timeline is entered not at 0800h as expected, but at 0800 minus the time shift, 2h in this instance, resulting in a zoomed graph entered at 0600h.  In this instance, the transient does not even show up in the graph because 0800h is outside the time range of the panel.

Shot 1
![screen shot 2016-08-30 at 12 27 30 pm](https://cloud.githubusercontent.com/assets/16888734/18102117/b451b684-6eae-11e6-9d56-cc675f8f05ea.png)

Shot 2
![screen shot 2016-08-30 at 12 27 36 pm](https://cloud.githubusercontent.com/assets/16888734/18102123/b922e96c-6eae-11e6-8f15-7cfe00032ff4.png)

Shot 3
![screen shot 2016-08-30 at 12 28 52 pm](https://cloud.githubusercontent.com/assets/16888734/18102129/bbffd74e-6eae-11e6-92de-d084cfdc02ba.png)

Shot 4
![screen shot 2016-08-30 at 12 29 01 pm](https://cloud.githubusercontent.com/assets/16888734/18102137/c11058bc-6eae-11e6-91b7-e752ed60ccbc.png)
